section3/exercises/arrays.js

```diff
@@ -96,21 +104,34 @@ YOU DO:
 Using the variables defined below, write a program that will tell a user if they
 will be able to call an Uber.
 
-The user can call an uber if they have 15% battery remaining, or more. In this case, it doesn't matter 
+The user can call an uber if they have 15% battery remaining, or more. In this case, it doesn't matter
 if the user has a charger at all, or what type.
 The can call an uber if they have a charger and it is a car charger.
 
 */
-var percentBatteryLeft = 12;
+var percentBatteryLeft = 100;
 var hasCharger = true;
 var chargerType = "car";
 
-// Write your conditional here
-
-
+// Conditional statement 'If' runs the log command when variables value meets the conditional operators evaluation
+if (percentBatteryLeft >= 15) {
+  console.log("Congrats! You can call an Uber!");
+// Second command executes if conditional operators in the second brackets evaluate to true
+} else if (percentBatteryLeft <= 14, hasCharger === true && chargerType === "car") {
```

Comment:
What if the `percentBatteryLeft = 14.5`? You should not assume you will always have round numbers.